### DISCUSSION TITLE
Fixing parameters to non-zero values?

### DISCUSSION THREAD

**Discussion by sojata** [cdriveraus/ctsem#47] [Q&A] [answered]
Dear All,
First, thanks to the developer for such a wonderful package for continuous time modeling. Your ability and skills to implement and maintain such a awesome package for us applied researchers evoke much respect. It is great to learn from guys like you.

Second I have a question about manually fixing parameters, specifically measurement errors in MANIFESTVAR to non-zero values. I have difficulties in getting stable estimates (i.e., the model fit and parameter depend on the used seed) when estimating a model with free parameters in the diagonal of the MANIFESTVAR matrix. When fixing the MANIFESTVAR matrix to zero, this problem disappears, of course, with the cost of a problematic assumption of no measurement error. As one mid-way solution, I wonder about fixing the diagonal of the MANIFESTVAR matrix manually to some more or less plausible value based on the reliability estimate of conducted longitudinal CFA. I tried that, and it gives me stable estimates.

However, I wonder about the logic of the ctsem in fixing the non-zero values. For example, if I want to fix the measurement error variance to (1-a)/a (as the diagonal of lambda is fixed to one), should I fix the diagonal of MANIFESTVAR to

(1-a)/a

or alternatively, to its standard deviation=

sqrt((1-a)/a)).

Or something else? If understand correctly the transformations used in ctsem, the standard deviation is the correct way, but I am not exactly sure.  Also, it seems that fixing higher values leads again to unstable parameters.

Thank you already beforehand if you find time to answer! I also attached the model specification if that help (first without and second with separate latent trends based on Driver and Tomasik (2022)).

[Model1 (1).pdf](https://github.com/cdriveraus/ctsem/files/11791246/Model1.1.pdf)
[Model2 (1).pdf](https://github.com/cdriveraus/ctsem/files/11791247/Model2.1.pdf)

best,
Jaakko Tammilehto

**Comment by cdriveraus** [comment, marked as answer]
Thanks for the glowing praise Jaako, it's enough that people are getting some use out of it for their own work :) :) you're correct that for fixed values they are specified in terms of the sd. I would urge you however to also try with the free measurement error parameters and consider reducing model complexity - sometimes there if nothing gained by including cross temporal effects once individual differences, measurement error, and correlated system noise are accounted for.

**Reply by sojata** [reply]
Dear Charles,

Sometimes your system just needs a right shock, and the whole world looks different (correlated system noise). Indeed, I have been so fascinated by the cross-effects that have not seen the forest from the trees. Thanks for opening my eyes!!!

best,
Jaakko
